cmake/Modules/AddCompilerRT.cmake

```diff
@@ -10,10 +10,18 @@ function(set_target_output_directories target output_dir)
   if(CMAKE_CONFIGURATION_TYPES)
     foreach(build_mode ${CMAKE_CONFIGURATION_TYPES})
       string(TOUPPER "${build_mode}" CONFIG_SUFFIX)
-      set_target_properties("${target}" PROPERTIES
-          "ARCHIVE_OUTPUT_DIRECTORY_${CONFIG_SUFFIX}" ${output_dir}
-          "LIBRARY_OUTPUT_DIRECTORY_${CONFIG_SUFFIX}" ${output_dir}
-          "RUNTIME_OUTPUT_DIRECTORY_${CONFIG_SUFFIX}" ${output_dir})
+	  if(CMAKE_CFG_INTDIR)
```

Comment:
This variable is always set to something:
* https://cmake.org/cmake/help/latest/variable/CMAKE_CFG_INTDIR.html

You probably mean this:
* http://cgold.readthedocs.io/en/latest/tutorials/build-types.html#detect-multi-single